conf/broker.conf

```diff
@@ -1097,6 +1097,18 @@ loadBalancerLoadSheddingStrategy=org.apache.pulsar.broker.loadbalance.impl.Overl
 # It only takes effect in the ThresholdShedder strategy.
 loadBalancerBrokerThresholdShedderPercentage=10
 
+# Message-rate percentage threshold between highest and least loaded brokers for
+# uniform load shedding. (eg: broker1 with 50K msgRate and broker2 with 30K msgRate
+# will have 66% msgRate difference and load balancer can unload bundles from broker-1
+# to broker-2)
+loadBalancerMsgRateDifferenceShedderThreshold=50
```

Comment:
IMHO, It's better to add a notice about how to enable uniform load shedding, like set `loadBalancerLoadSheddingStrategy="org.apache.pulsar.broker.loadbalance.impl.UniformLoadShedder"`

Reply:
+1

@rdhabalia 
For the doc side, how about adding docs like below? (feel free to correct me if my understanding is wrong)
![image](https://user-images.githubusercontent.com/50226895/142970058-b36bb388-74bc-41e6-88f5-d9865427ffcd.png)
That is:
- Add a section `Shed load manually` 
- 3 strategies (`ThresholdShedder`, `OverloadShedder`, and `UniformLoadShedder`) belong to `Shed load manually` section
- Add explanations of "when to use" and "how to use" for each strategy

If so, can you add them? Thanks!

Reply:
added into documentation